content/blog/2023/2023-08-31-fairness-hiding-protected-attribute.md

```diff
@@ -0,0 +1,56 @@
++++
+author = "Žan Mervič"
+date = "2023-08-31"
+draft = false
+title = "Orange Fairness - Hiding Protected Attribute"
```

Comment:
Title is weak. This is very general post (which I like) and more conceptual than Orange-related. So the title should reflect this. Maybe: "Does removing features remove bias?"

Also, we usually avoid the word "attribute" when addressing non-specialist audience.